README.md

```diff
@@ -42,7 +42,7 @@ Contributions, issues, and feature requests are welcome!
 
```

Comment:
- [ ] I think you forget about recoding a video for the documentation so please acoording to the requirements record a video and add the link to you PR 🔥🚀🚀

![image](https://user-images.githubusercontent.com/58553711/190172744-deddf68f-8f75-4016-8a69-80f4d4ca8987.png)